tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/export-deployment-artifacts/docker-config-dialog.js

```diff
@@ -61,10 +62,15 @@ define(['require', 'lodash', 'jquery'],
                 var password = self.container.find("#password").val();
                 var email = self.container.find("#email").val();
                 var pushDocker = self.container.find("#docker-push-checkbox").is(":checked");
-                var missingDockerConfigErrorMessage = self.container.find("#missing-docker-config-error-message");
                 if (pushDocker) {
-                    if (imageName != "" && userName != "" && password != "" && email != "") {
-                        missingDockerConfigErrorMessage.hide();
+                    var upperCase = new RegExp('[A-Z]');
+                    if (!imageName.match(upperCase)) {
+                        self.dockerNameInUpperError.hide();
+                    } else {
+                        self.dockerNameInUpperError.show();
+                    }
+                    if (imageName !== "" && userName !== "" && password !== "" && email !== "") {
```

Comment:
Are we trim the text before above validation?

Reply:
Fixed in here https://github.com/siddhi-io/distribution/pull/752/commits/90bb44445c41a71bd18f5efe9b3338d5358fc112